sanguk/app.js

```diff
@@ -33,20 +33,77 @@ app.get("/ping", (req,res)=>{
 })
 
 app.post("/users/signup", async (req,res,next)=>{
-    const {username, password, email, phonenumber } =req.body
+    const {username, userProfileImage } =req.body
 
     await myDataSource.query(
         `INSERT INTO users(
             username,
-            password,
-            email,
-            phonenumber
-        ) VALUES(?, ?, ?, ?)`
-        ,[username, password, email, phonenumber]
+            userProfileImage
+        ) VALUES(?, ?)`
+        ,[username, userProfileImage]
     )
     res.status(201).json({message : "userCreated"})
 })
 
+app.post("/posts/signup", async (req,res,next)=>{
+    const {postingId, postingImageUrl, postingContent} =req.body
+
+    await myDataSource.query(
+        `INSERT INTO posts(
+            postingId,
+            postingImageUrl,
+            postingContent
+        ) VALUES(?, ?, ?)`
+        ,[postingId, postingImageUrl, postingContent]
+    )
+    res.status(201).json({message : "postCreated"})
+})
+
+app.get ('/search/everything',async (req,res)=>{
```

Comment:
```suggestion
app.get ('/posts',async (req,res)=>{
```
위에서 한차례 리뷰 남겨드린 것과 비슷한 내용이에요 상욱님! 해당 API는 백엔드 데이터베이스 서버에 있는 모든 post data들을 불러오는 API이지요? 그렇다면 http method중 data를 가져온다는 의미를 지닌 get 과 함께 `app.get("/posts", . . .` 라고만 작성되어도 무엇을(what)? posts를, 어떻게(how)? get 과 같이 동사와 목적어의 조합으로 알아보기 쉽게 나타낼 수 있습니다.

이렇듯 자명한 (그 자체만으로도 스스로 설명이 되는) end-point로 이어진 API를 작성하는 것을 RESTful한 API라고 하는데 이는 금주에 있는 세션을 통해 더 디테일하게 배우고 다룰 예정이니 이러한 느낌이구나~ 만 가져가시면 될 것 같습니다-! 수정해주세요!